Tests/DataXScenarios/DataXScenarios/Helper.cs

```diff
@@ -21,5 +21,15 @@ public string GetInitializeKernelJson(ScenarioContext context)
         {
             return $"{{\"name\": \"{context[Context.FlowName] as string}\", \"userName\": \"{context[Context.FlowName] as string}\", \"eventhubConnectionString\": \"{context[Context.EventhubConnectionString] as string}\", \"eventHubNames\": \"{context[Context.EventHubName] as string}\", \"inputType\": \"iothub\", \"inputSchema\": {context[Context.InputSchema] as string}, \"kernelId\": \"{context[Context.KernelId] as string}\", \"normalizationSnippet\": {context[Context.NormalizationSnippet] as string}, \"databricksToken\": \"{context[Context.DataBricksToken] as string}\"}}";
```

Comment:
This shouldn't be included in InitializeKernel payload.
\"databricksToken\": \"{context[Context.DataBricksToken].

Reply:
DataBricks token key is removed now. The references to DataBricksToken from other methods were also removed.